programs/vote/src/vote_state.rs

```diff
@@ -334,6 +343,26 @@ impl VoteState {
             }
         }
     }
+
+    fn check_timestamp(&self, slot: Slot, timestamp: UnixTimestamp) -> Result<(), VoteError> {
```

Comment:
can you just fold this into `process_timestamp()` ?

Reply:
Yep. I thought it might help with unit testing, but it is unnecessarily verbose.